automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala

```diff
@@ -622,7 +622,7 @@ class RawlsApiSpec extends TestKit(ActorSystem("MySpec")) with FreeSpecLike with
             register cleanUp Rawls.submissions.abortSubmission(projectName, workspaceName, submissionId)
 
             val submissionPatience = PatienceConfig(
-              timeout = scaled(Span(16, Minutes)),
+              timeout = scaled(Span(36, Minutes)),
```

Comment:
Why 36?

Reply:
See the discussion linked from the ticket. Spoiler alert: I guessed.